src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -184,6 +201,7 @@ class CodyAutocompleteManager {
       triggerKind: InlineCompletionTriggerKind,
       cancellationToken: CancellationToken,
       lookupString: String?,
```

Comment:
I looked quickly into the code. 

It seems that it would be possible to disable autocompletion for some specific lookup elements. Now, we are depending directly on the list's selected values (strings). Instead, we could try to check for a precise `com.intellij.codeInsight.lookup.LookupElement` implementation and decide.

If think that would be a nice improvement, b/c the autocompletions for the actions look a bit dumb 😅 I guess we can provide it later. 

What we have here, LGTM.